conf/base.config

```diff
@@ -22,14 +22,137 @@ process {
   maxErrors = '-1'
 
   // Process-specific resource requirements
-  withName: fastqc {
-    errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
+
+  withName:combine_public_annotation {
```

Comment:
Looks like you have 2-3 configurations in general. You could use the 

`withLabel:highmem` directive to define one resource scope for multiple processes. 

https://www.nextflow.io/docs/latest/config.html#process-selectors